be/src/storage/segment_replicate_executor.cpp

```diff
@@ -142,11 +149,15 @@ void ReplicateChannel::_send_request(SegmentPB* segment, butil::IOBuf& data, boo
     _closure->ref();
     _closure->reset();
     _closure->cntl.set_timeout_ms(_opt->timeout_ms);
+    _closure->cntl.ignore_eovercrowded();
```

Comment:
Will it consume large memory if ignore overcrowed?

Reply:
These memory will be limited by `load_process_max_memory_limit_bytes`, which was not the case before, so the total amount of memory used will be even less than before. If load_memory is full, the OlapTableSink will be backpressured, which is also reasonable.